mmdet/models/layers/transformer/dino_layers.py

```diff
@@ -54,9 +54,19 @@ def forward(self, query: Tensor, value: Tensor, key_padding_mask: Tensor,
                 regression results.
 
         Returns:
-            Tensor: Output queries of Transformer encoder, which is also
-            called 'encoder output embeddings' or 'memory', has shape
+            Tensor: Output queries of Transformer decoder, which is also
```

Comment:
Return type seems to be `Tuple[Tensor, Tensor]`, not `Tensor`.
Please refer to other docstrings such as those in `DeformableDetrTransnformerDecoder`.

Besides, `Output queries` --> `Output queries and references`